scripts/cangen/main.py

```diff
@@ -68,6 +68,14 @@ def parse():
     bus_list = config["canGen"]["busses"]
     output_path = config["canGen"].get("outputPath", DEFAULT_OUTPUT_DIR)
 
+
+#clear the directory file
```

Comment:
Can you improve this comment? Something about getting rid of old files before making new ones